js/security-headers.js

```diff
@@ -0,0 +1,33 @@
+import env from "../config/env.generated.json";
+
+export default {
+  directives: {
+    defaultSrc: [
+      `'none'`
+    ],
+    scriptSrc: [
+      `'self'`,
+      `'unsafe-inline'`
+    ],
+    fontSrc: [
+      `'self'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `'self'`,
+      `'unsafe-inline'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `'self'`,
+      `data:`,
+      `*`
```

Comment:
hmm, does this make the previous two lines irrelevant... the MDN docs are not super clearn, would you know @cadecairos ?